install-general.md

```diff
@@ -362,25 +362,32 @@ To install the Tanzu CLI on a Windows operating system:
     ```
     tanzu version
     ```
+## Update the Tanzu CLI
 
-## Install the Tanzu CLI Plugins
-
-After you have installed the tanzu core executable, you must install package, imagepullsecret, apps and app-accelerator CLI plugins.
-
-1. (Optional) Remove existing plugins from any previous CLI installations.
+If you have any previous Tanzu CLI installed please run below steps before moving on to [Install the Tanzu CLI Plugins](#Install the Tanzu CLI Plugins).
+1.  Set TANZU_CLI_NO_INIT=true
+2.  export TANZU_CLI_NO_INIT=true
+3.  Run this command to make sure the default plugin repo points to the right path 
+    ```
+    tanzu plugin repo update -b tanzu-cli-framework core
+    ```
 
+4.  Remove existing plugins from any previous CLI installations.
```

Comment:
```suggestion
1.  Set the environment variable `TANZU_CLI_NO_INIT`.
     ```
     export TANZU_CLI_NO_INIT=true
     ```

2.  Run this command to make sure the default plugin repo points to the right path:
    ```
    tanzu plugin repo update -b tanzu-cli-framework core
    ```

3.  Remove existing plugins from any previous CLI installations.

Reply:
@jncd  Please merge it now. I have resolved all the conflicts. I think conflicts were due to some PRs merged before mine.